src/app.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Main } from './components/Main';
+import { Main } from './Components/main';
```

Comment:
Probably "Main" since it's also a React component.

Reply:
Name was changed.